### PR TITLE
Various UI tweaks

### PR DESCRIPTION
**Pull request by lintool**
Major change: changed metadata display from table (which overflowed width poorly) to header/paragraph combinations.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -44,13 +44,13 @@ for (let i = 0; i < datasets.length; i++) {
...
                         '<h6 class="mb-0">' + 
-                            `<a class="collapsed text-left stretched-link" type="button" data-target="#collapse-${i}" aria-expanded="false" aria-controls="collapse-${i}" onclick="toggleMetadata(${i}, false);">` +
+                            `<a class="collapsed text-left stretched-link" data-target="#collapse-${i}" aria-expanded="false" aria-controls="collapse-${i}" onclick="toggleMetadata(${i}, false);">` +
                                 `${title}` + 
                             '</a>' +
                         '</h6>' +
                         '<div class="row justify-content-end">' +
-                            `<button id="showBounds-${i}" class="btn btn-link" onclick="toggleBounds(${i});" style="display: none; z-index: 2000;">Show Bounds</button>` +
-                            `<button id="hideBounds-${i}" class="btn btn-link" onclick="toggleBounds(${i});" style="z-index: 2000;">Hide Bounds</button>` +                        
+                            `<button id="showBounds-${i}" class="btn btn-link" onclick="toggleBounds(${i});" style="display: none; z-index: 2000; font-size: 50%;">[Show Bounds]</button>` +
```
Comment (gauchm) [inline index.js]: grafik
I think that's too small, I can barely read it. Seems like I either need a screen with more DPI or new glasses... :)

File: index.html [hunk 2]
```diff
@@ -13,10 +13,10 @@
     </head>
     <body>
         <header class="navbar navbar-expand sticky-top navbar-dark bg-info flex-column flex-md-row" id="header">
+            <img class="img-right mr-2" src="GWF_logo.png" height="30"/>
```
Comment (gauchm) [inline index.html]: The logos seem pretty small to me now. The "GWF" is hardly readable on my screen. 36 would be readable.

File: index.js [hunk 3]
```diff
@@ -44,13 +44,13 @@ for (let i = 0; i < datasets.length; i++) {
     var card =  `<div class="card" id="dataset-${i}">` +
                     `<div class="card-header" id="heading-${i}">` +
                         '<h6 class="mb-0">' + 
-                            `<a class="collapsed text-left stretched-link" type="button" data-target="#collapse-${i}" aria-expanded="false" aria-controls="collapse-${i}" onclick="toggleMetadata(${i}, false);">` +
+                            `<a class="collapsed text-left stretched-link" data-target="#collapse-${i}" aria-expanded="false" aria-controls="collapse-${i}" onclick="toggleMetadata(${i}, false);">` +
```
Comment (gauchm) [inline index.js:47]: Thanks for catching that! In Firefox it doesn't change anything, but chrome looks horrible with `type="button"`!

File: index.js (hunk 1 above)
Comment (lintool) [reply index.js]: Made larger for now. But eventually I'd like this replaced with some icon?

File: index.html (hunk 2 above)
Comment (lintool) [reply index.html]: Addressed.

**Comment by lintool** [comment]
Addressed CR.

**Review by gauchm — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (gauchm) [reply index.js]: Maybe the Location-marker icon? (Such as https://material.io/tools/icons/?icon=location_on&style=baseline and https://material.io/tools/icons/?icon=location_off&style=baseline)

Comment (lintool) [reply index.js]: 👍 

Very nice.

**Pull request merged**